### PR TITLE
docs for bridger

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

File: content/zh-CN/wiki-us-bridgechain.md [hunk 1]
```diff
@@ -6,7 +6,69 @@ sidebar_label: 跨链转接桥网络
 
 达尔文桥接链是达尔文网络中最重要的部分，也是各个应用子链和公链的桥接枢纽。
 
-达尔文网络自身可以作为一个独立的跨链网络运行，达尔文链将负责共识安全和跨链互操作。同时，得益于 Polkadot 提供了一套开放的平行链接入方式，达尔文链也可以选择接入 Polkadot 作为平行链运行，Polkadot 将接管并负责达尔文链的安全，这样，达尔文网络中的所有应用链将可以通过 Polkadot 连接至外部更广泛的区块链网路。
-.
+达尔文网络自身可以作为一个独立的跨链网络运行，达尔文链将负责共识安全和跨链互操作。同时，得益于 Polkadot 提供了一套开放的平行链接入方式，达尔文链也可以选择接入 Polkadot 作为平行链运行，Polkadot 将接管并负责达尔文链的安全，这样，达尔文网络中的所有应用链将可以通过 Polkadot 连接至外部更广泛的区块链网络。
 
-TODO: Bridge Chain image and more description
+达尔文的跨链转接桥协议采用的是CBA(Cryptocurrency-backed Assets)的模型，底层使用chain relay证明交易的存在性以及区块共识。
```
Comment (boundless-forest) [inline content/zh-CN/wiki-us-bridgechain.md]: `CBA(Cryptocurrency-backed Assets)` 两边预留空格，`chain relay` 同样。

File: content/zh-CN/wiki-us-bridgechain.md [hunk 2]
```diff
@@ -6,7 +6,69 @@ sidebar_label: 跨链转接桥网络
 
 达尔文桥接链是达尔文网络中最重要的部分，也是各个应用子链和公链的桥接枢纽。
```
Comment (wuminzhe) [inline content/zh-CN/wiki-us-bridgechain.md:7]: 英文版本文档

File: content/zh-CN/wiki-us-bridgechain.md (hunk 1 above)
Comment (wi1dcard) [reply content/zh-CN/wiki-us-bridgechain.md]: @AsceticBear  哈哈哈哈，跟我学坏了！

**Comment by hackfisher** [comment]
@wuminzhe 存在约定，需要先提供英文文档，然后再统一做翻译和国际化工作吗？

**Comment by wuminzhe** [comment]
> @wuminzhe 存在约定，需要先提供英文文档，然后再统一做翻译和国际化工作吗？

是这么考虑的，至少要有英文版本的

**Comment by hackfisher** [comment]
@xiaoch05 Any update for this?

Do you think the convention of English first requirement make sense, or too strict for current stage?

**Comment by xiaoch05** [comment]
> @xiaoch05 Any update for this?
> 
> Do you think the convention of English first requirement make sense, or too strict for current stage?

this need an English version. I will add it soon.

**Review by robinwei-itering — APPROVED** [review]

**Pull request merged**